lib/plugins/create/templates/aws-csharp/build.cmd

```diff
@@ -0,0 +1,3 @@
+dotnet restore
+dotnet publish -c release
+dotnet lambda package --configuration release --framework netcoreapp1.0 --output-package bin/release/netcoreapp1.0/deploy-package.zip
```

Comment:
Just confirmed, we can get rid of `dotnet publish` since `lambda package` does the building, publishing *and* zipping.